src/test/java/games/strategy/triplea/settings/ResetFunctionTest.java

```diff
@@ -0,0 +1,38 @@
+package games.strategy.triplea.settings;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.Arrays;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.hamcrest.core.Is;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
We've recently switched all Mockito test runners to `MockitoJUnitRunner.StrictStrubs.class`.